src/Microsoft.DocAsCode.Common/Loggers/WarningLogListener.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Common
+{
+    using System;
+    using System.Collections.Concurrent;
+
+    public sealed class WarningLogListener : ILoggerListener
```

Comment:
how about LogWithSpecifiedLevelListener?

---
In reply to: [76367221](https://github.com/dotnet/docfx/pull/638#discussion_r76367221) [](ancestors = 76367221)